catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -126,7 +136,30 @@ private CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
                 filterBuilder);
 
         Stopwatch stopwatch = Stopwatch.createStarted();
-        QueryResponse response = catalogFramework.query(request);
+        QueryResponse response = null;
```

Comment:
✏️  We might want to refactor all this new block into a method that returns the overall `response`